### PR TITLE
Fix Password Storing And Entering On Linux

### PR DESCRIPTION
**Pull request by jimmon89**
Add changes by @chtheiss for password storing and entry. Add fixes to load different keystore method based on OS. Remove support for Mac OSX completely.
Add password entry functions to linux launch method. Create a python requirements file requirements_linux.txt Modify .github/workflows/linux_build.yml to point to requirements_linux.txt

Too tired to note rest, live with it
Brain broken, oog good at sun
ask words then

### CODE REVIEW

File: .github/workflows/linux_build.yml [hunk 1]
```diff
@@ -40,18 +40,16 @@ jobs:
...
           python -m pip install --upgrade pip
-          pip install -r requirements.txt
+          pip install -r requirements_linux.txt
           pip install nuitka
           conda install libpython-static -y
       
       - name: Build Executable
         run: |
-          python -m nuitka --standalone --enable-plugin=pyside6 --disable-console --onefile --include-data-dir=images=images --output-dir=build duskhaven_launcher.py
+          python -m nuitka --standalone --enable-plugin=pyside6 --disable-console --onefile --include-data-dir=images=images --output-dir=dist  duskhaven_launcher.py
```
Comment (chtheiss) [inline .github/workflows/linux_build.yml]: this needs to be --output-dir=build

File: README.md [hunk 2]
```diff
@@ -117,6 +125,7 @@ winetricks win10 ie8 corefonts dotnet48 vcrun2019 dxvk
 ```
 cd "/home/$USER/.local/share/Duskhaven/"
 wget https://github.com/chtheiss/duskhaven_launcher/releases/latest/download/duskhaven_launcher.bin
+wget https://github.com/chtheiss/duskhaven_launcher/blob/main/images/Duskhaven.png
```
Comment (chtheiss) [inline README.md]: Why do you need to do this?

Comment (chtheiss) [reply README.md]: Ahh it's the ico. maybe rename it.

Comment (jimmon89) [reply README.md]: to download the icon for the dekstop shortcut on linux
if you prefer it can be removed

**Pull request closed without merging**